parser/ast/ddl.go

```diff
@@ -259,7 +259,9 @@ type IndexPartSpecification struct {
 
 	Column *ColumnName
 	Length int
-	Expr   ExprNode
+	// Desc should be ignored because TiDB not support index sort direction.
```

Comment:
Better to keep the claim that TiDB is designed to behave consistently with MySQL like before.

Reply:
TiDB parser wants to maintain maximum compatibility with MySQL. So it's better to define this struct in Go. So do you think we should change a comment?

Reply:
Yeah, I mean it's better to retain the original comment "// Order is parsed but just ignored as MySQL did."

Reply:
Hey @JmPotato , I think this comment had overdue.
MySQL parser parses order index in v5.x, but it will be ignored because MySQL doesn't support it in v5.x. But since MySQL 8.0, MySQL will not ignore it because it had already been supported.
I prefer to use `// Order is parsed but should be ignored because TiDB doesn't support it.`, how about you?

Reply:
Since TiDB claims to be compatible with MySQL v5.7 according to https://docs.pingcap.com/tidb/dev/mysql-compatibility, so I think it's OK to level the original comment. But actually, my core idea is that the parser is some kind of a public dependency, and we can probably avoid introducing things like "TiDB" into it as much as possible. Using "Order is parsed but should be ignored because MySQL v5.7 doesn't support it" sounds more natural to me. WDYT?

Reply:
LGTM! I will modify it.

Reply:
Done. PTAL @JmPotato